app/services/integrations/aggregator/taxes/invoices/payload.rb

```diff
@@ -47,7 +47,7 @@ def fee_item(fee)
             mapped_item ||= empty_struct
 
             {
-              'item_id' => fee.id,
+              'item_id' => fee.id || fee.item_id,
```

Comment:
`|| fee.item_id` part is still needed for one-off part where taxes are applied on the UI form (unrelated to invoice generation). In that case, `item_id` is identifier which UI uses to find matching add_on in the form (add_od_id is familiar data on FE side). This issue is also regression so `fee.id || fee.item_id` covers all cases now without any FE adjustments and bigger refactorings